db/migrations/20230102023729_create_users_table.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  email VARCHAR(200) NOT NULL,
+  password VARCHAR(200) NOT NULL,
+  name VARCHAR(30) NOT NULL,
+  phone_number VARCHAR(30) NOT NULL,
+  address VARCHAR(200) NULL,
+  point INT NOT NULL DEFAULT 0,
```

Comment:
```suggestion
  point INT NOT NULL DEFAULT 0,
```
intger field가 아닌 decimal로 사용해주세요. close된 PR에서 남겨드린 코멘트 누락없이 모두 반영해주세요!